### PR TITLE
New Data Structure Operations cheat sheet

### PR DESCRIPTION
**Pull request by navneetzz**
**What does your Instant Answer do?**
My Instant answer provide reference to Big-O complexities of Data Structure Operations.

dsa

---
https://duck.co/ia/view/data_structure_operations_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Data Structure Operations Cheat Sheet](https://duck.co/ia/view/data_structure_operations_cheat_sheet)

**Description**: Summary of Big-O time complexities for Data Structure Operations

**Example Query**: [data structure operations cheat sheet](https://beta.duckduckgo.com/?q=data structure operations cheat sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/data_structure_operations_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/data_structure_operations_cheat_sheet)*

**Comment by GuiltyDolphin** [comment]
@navneetzz We don't need the template answer anymore! Just link to the IA page (and maybe provide a brief summary).

**Comment by navneetzz** [comment]
@GuiltyDolphin Done

File: share/goodie/cheat_sheets/json/data-structure-operations.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
+{
+    "id": "data_structure_operations_cheat_sheet",
+    "name": "Data Structure Operations",
+    "description": "Summary of Big-O time complexities for Data Structure Operations",
+    "metadata": {
+        "sourceName": "Wikipedia",
+        "sourceUrl": "https://en.wikipedia.org/wiki/List_of_data_structures"
+    },
+    "aliases": [
+        "data structure operation", "data structure"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/data-structure-operations.json]: I'm not sure `data structure` is an appropriate trigger -- I would expect a "data structure cheat sheet" that describes common data structures

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=data structure operations cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/data_structure_operations_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=data structure operations cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/data_structure_operations_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@navneetzz Thanks! This LGTM :+1:

**Pull request merged**